astroquery/mast/observations.py

```diff
@@ -302,7 +302,7 @@ def query_criteria_async(self, pagesize=None, page=None, **criteria):
 
         return self._portal_api_connection.service_request_async(service, params)
 
-    def query_region_count(self, coordinates, radius=0.2*u.deg, pagesize=None, page=None):
+    def query_region_count(self, radius=0.2*u.deg, pagesize=None, page=None, *, coordinates):
```

Comment:
no, this syntax won't work, as well as introduces an API change. Keep it as: mandatory arguments, that can be positional, and then force the optional keyword arguments as keyword only. 
(Same logic for the rest of the lines, I don't repeat it there)

```suggestion
    def query_region_count(self, coordinates, *, radius=0.2*u.deg, pagesize=None, page=None):
```